tests/legacy-cli/e2e/tests/test/test-sourcemap.ts

```diff
@@ -1,29 +1,37 @@
 import { writeFile } from '../../utils/fs';
-import { execAndWaitForOutputToMatch, killAllProcesses } from '../../utils/process';
+import { execAndWaitForOutputToMatch } from '../../utils/process';
 import { updateJsonFile } from '../../utils/project';
 
 export default async function () {
+  await writeFile('src/app/app.component.spec.ts', `
+      it('show fail', () => {
+        expect(undefined).toBeTruthy();
+      });
+    `);
+
   await updateJsonFile('angular.json', configJson => {
     const appArchitect = configJson.projects['test-project'].architect;
     appArchitect.test.options.sourceMap = {
       scripts: true,
     };
   });
 
-  await writeFile('src/app/app.component.spec.ts', `
-      it('show fail', () => {
-        expect(undefined).toBeTruthy();
-      });
-    `);
+  // when sourcemaps are not working the stacktrace won't point to the spec.ts file.
+  await execAndWaitForOutputToMatch(
```

Comment:
Is this the right function to use here? We usually use this one for watching processes. But I think the `ng` wrapper has a way to get the output too.

Reply:
Changed this to 
```
  try {
    await ng('test', '--watch', 'false');
    throw new Error('ng test should have failed.');
  } catch (error) {
    if (!error.message.includes('app.component.spec.ts')) {
      throw error;
    };
  }
```